conda/gateways/connection/session.py

```diff
@@ -156,6 +157,26 @@ def __init__(self, auth: AuthBase | tuple[str, str] | None = None):
 
         self.proxies.update(context.proxy_servers)
 
+        ssl_context = None
+        if context.ssl_verify == "truststore":
+            try:
+                import ssl
+
+                import truststore
+
+                ssl_context = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+            except ImportError:
+                warnings.warn(
+                    dals(
+                        """
+                        The `ssl_verify: truststore` setting is only supported on python >=3.10.
+                        Falling back to `ssl_verify: true`
```

Comment:
This is misleading, it doesn't fall back to `ssl_verify = True` but to the default of the parent class' `verify` default value.